ProcessLib/NeumannBc.h

```diff
@@ -111,7 +116,7 @@ class NeumannBc
         LocalAssemblerBuilder local_asm_builder(
             initializer, *_local_to_global_index_map);
 
-        auto elementValueLookup = [this](MeshLib::Element const& e)
+        auto elementValueLookup = [this](MeshLib::Element const&)
```

Comment:
General interface.
When we start to use material ids the elementValueLookup will be dependent on the element itself. For the constant value for any of elements the argument is not used.